### ISSUE TITLE
Crashes.trackError(Exception ex) for Android

### ISSUE THREAD

**Issue by Remcoj** [microsoft/appcenter#827]
We would like the ability to send exceptions from Native Android (java) to Appcenter using the Crashes.trackError method.
According to the documentation [https://docs.microsoft.com/en-us/appcenter/diagnostics/limitations#handled-errors](url) this is not possible.

In some situations we do not want to crash the app to get the error. Therefore we suround the code with a try/crash statement. Even though this prevents the app from crashing we would like to know if this error occured so we can improve the stability of the app.

In theory with feature #31 we could send it manually, however we would prefer to use the Appcenter SDK.

**Comment by herbig** [comment]
Hi @winnieli1 ! Do you have an ETA yet on this?  We'd like this feature before migrating Pocket on Android from Hockey to App Center.

**Comment by pengcao1** [comment]
any updae?

**Comment by winnieli1** [comment]
Yep! We added this to our roadmap and expect to complete this in the upcoming months.

**Comment by winnieli1** [comment]
Hi all! We just released this with the Android SDK Version 2.5.0.
You can find the docs here: https://docs.microsoft.com/en-us/appcenter/sdk/crashes/android#handled-errors.

**Comment by danielwilson1702** [comment]
Just wondering, are these supposed to show up in Diagnostics -> Issues -> Errors? I am not getting any and I can see them appear in the Log Flow screen:

c776a1a0
HANDLEDERROR - {"error_id":"720404aa-ffcb-468c-b031-2dcc48a17477"}

I will keep trying maybe it needs time or I shouldn't test on an emulator

**Comment by winnie** [comment]
Hi @danielwilson1702 - yep, the handled exceptions should appear in the Diagnostics -> Issues -> All and Diagnostics -> Issues -> Errors tab. 

I'm not aware of any relevant limitations regarding emulators. Were you able to get your errors to show up? If not, please start a support thread (blue chat bubble on the bottom right) so our support and engineering team can take a closer look. Thanks!

**Comment by danielwilson1702** [comment]
Thanks @winnie , yeah I have an on-going chat with the support but no answer yet. 

So about 48 hours after I tried to log many handled errors, I ended up getting just a single instance in the dashboard. Then I tried maybe a couple dozen more times, and 5 more appeared what looks like over 24 hours later (again missing many). 

Maybe it makes a difference if I am using a debug apk vs a production one? Just need to understand why so many are missing, and what kind of delays to expect really. I also noticed delays (24 hours I think) in `generateTestCrash` crashes appearing so I don't know if that's related too, I assume so. Will keep trying to see if I can pin down some behavior.
